app/models/pacbio/run.rb

```diff
@@ -53,7 +53,15 @@ def comments
     # using pipelines.yml configuration to generate data
     def generate_sample_sheet
       configuration = pacbio_run_sample_sheet_config
-      sample_sheet = RunCsv::PacbioSampleSheet.new(object: self, configuration:)
+      # matches CSV Version in sample sheets
+      sample_sheet_class =
```

Comment:
This is not quite what I was suggesting. I was thinking more along the lines of adding the SampleSheet class to the configuration and then constantising it here. Happy to go through what I mean.

Reply:
I still think there might be a misunderstanding. The change I am suggesting is just adding the Sample class name to the configuration. It is a simple thing to do.

Reply:
Good enough for now! When we remove the feature flag we could add the sample sheet class to all of the versions maybe? I think the sample sheet class is a good addition and so we could use it when there are further changes. How does that sound? I think it could also be a way of making samples sheets more flexible across technologies.